zebra-network/src/protocol/external/message.rs

```diff
@@ -307,3 +308,138 @@ pub enum RejectReason {
     Checkpoint = 0x43,
     Other = 0x50,
 }
+
+// Modified from the derived Debug, to summarise `Vec`s
+impl fmt::Debug for Message {
```

Comment:
We should probably clean this up to not use the same naming and style as the derive macro does.

* Get rid of all of the `__ident`s
* Chain our `debug_struct` builders rather than doing bindings and discarding intermediate references.
* remove the unnecessary `&&(*ident)` stuff which is probably some auto deref nonsense we don't need to deal with

Reply:
Fixed in 4d8bbd54badcbcece11c1ca186ee4cc4cec60b88